Server/src/main/java/Database/TheAuPair/Models/Activity.java

```diff
@@ -0,0 +1,162 @@
+package Database.TheAuPair.Models;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+@Document("Activities")
+public class Activity
+{
+    @Id
+    private String id;
+
+    @Field ("name")
+    private String name;
+    @Field ("description")
+    private String description;
+    @Field ("location")
+    private String loc;
+    @Field ("time_start")
```

Comment:
Should we not make this CamelCase to keep consistency? Unless it's generated by the DB or something?